frontend/packages/dev-console/src/components/import/deployImage-submit-utils.ts

```diff
@@ -307,7 +309,10 @@ export const createOrUpdateDeploymentConfig = (
   };
 
   const deploymentConfig = mergeData(originalDeploymentConfig, newDeploymentConfig);
-
+  if (verb === 'update') {
+    deploymentConfig.spec.template.spec.containers =
+      newDeploymentConfig.spec.template.spec.containers;
```

Comment:
Can we put this in ```mergeData``` function?

Reply:
Having this one outside `mergeData` because it is overwritting the old values(envs) with new values(envs) and not merging them.

Reply:
@karthikjeeyar if you see the `mergeData` function, its merging the two resources and also  after that overriding the old labels with new. This use-case is similar, so it makes sense to move this to `mergeData`.

Reply:
updated it